test/manual/amp-lightbox-gallery-launch.amp.html

```diff
@@ -71,6 +81,18 @@ <h1>Lightbox Carousel Test</h1>
       width="400"
       height="300"
       aria-labelledby="my-aria-label"></amp-img>
+  </amp-carousel>
+
+  <h1>Lightbox Carousel with Video Test</h1>
+  <amp-carousel height="300"
+  width="400"
+  layout="fixed"
+  type="slides"
+  lightbox>
+    <amp-img src="https://images.unsplash.com/photo-1503843778847-2b8bdce2ed3d"
+      width="400"
+      height="300"
+      alt="a beautiful cat"></amp-img>
     <amp-video
       src="https://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ForBiggerJoyrides.mp4"
       width="358"
```

Comment:
width and height not needed for layout fill

Reply:
I think this was changed to `layout="responsive"` instead.